config/zones.json

```diff
@@ -3750,8 +3750,26 @@
     "timezone": null
   },
   "US-HI": {
+	"_comment": "biomass includes 68.5 MW of Waste-to-energy and 188 MW of biodiesel.",
+	"capacity": {
+		"biomass": 256.5,
+		"coal": 0,
+		"gas": 0,
+		"geothermal": 0,
+		"hydro": 0,
+		"oil": 0,
+		"solar": 781,
+		"wind": 171,
+		"unknown": 0
+	},
+	"contributors": [
```

Comment:
can you fix indentation to be consistent? thanks!

Reply:
Done!